team-norms/bau_expectations.md

```diff
@@ -0,0 +1,20 @@
+# Business As Usual (BAU) Expectations
+
+Teams must deliver software systems in accordance with our Business As Usual Expectations that make software systems easier to run, debug, and improve.
+
+The Made Tech Support Team requirements are identified below. Where the application is transitioning to a customer support team, you should liaise with the customer team to understand their specific needs.
+
+## Application Requirements
+
+- Centralised Logging (Papertrail)
+- Application Performance Monitoring (Datadog)
```

Comment:
Unsure if we can fully go with this _currently_. I'm pretty sure when we started using Datadog for infra, the APM was free. Now they're charging for it too separate.

Reply:
Yeah agree - we should get APM into projects as standard but all the options are fairly pricy.

Reply:
@davidwinter did we figure out New Relic was going to do the job we need on the free tier for now?